src/Optimizer/Transformer/ServerSideRendering.php

```diff
@@ -605,7 +605,7 @@ private function createResponsiveSizer(
         $style = 'padding-top:%s%%'
     ) {
         $padding       = $height->getNumeral() / $width->getNumeral() * 100;
-        $paddingString = (string) round($padding, 4);
+        $paddingString = (string) number_format($padding, 4, '.');
```

Comment:
You need to provide the fourth argument as well here.
See https://3v4l.org/clDrF

```suggestion
        $paddingString = (string) number_format($padding, 4, '.', '');
```